<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWidget</class>
 <widget class="QWidget" name="GraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sampling interval:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="samplingInterval">
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Buffer size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="bufferExp">
     <property name="prefix">
      <string>10^</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mean sampling frequency (Hz):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="cycleFreq">
     <property name="text">
      <string>0000</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="active">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QTableWidget" name="table">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>400</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Plotting with QCustomPlot, licensed under GPLv3. More info: http://www.qcustomplot.com/</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
